Composer/cypress/support/commands.ts

```diff
@@ -19,6 +19,43 @@ Cypress.Commands.add('createBot', (botId: string, botName?: string) => {
   });
 });
 
+Cypress.Commands.add('createBotV2', (botName: string, callback: (bot: any) => void) => {
+  const params = {
+    description: '',
+    location: '',
+    name: `__Test${botName}`,
+    runtimeLanguage: 'dotnet',
+    runtimeType: 'webapp',
+    schemaUrl: '',
+    storageId: 'default',
+    templateId: '@microsoft/generator-bot-empty',
+    templateVersion: '1.0.0-rc3',
+  };
+
+  const pollingRequestBotStatus = (jobId: string, callback: (result: any) => void) => {
+    cy.wait(2000);
+    try {
+      cy.request('get', `/api/status/${jobId}`).then((res) => {
+        const { httpStatusCode, id, result } = res.body;
+        if (httpStatusCode !== 200) {
+          pollingRequestBotStatus(id, callback);
+        } else {
+          callback(result);
+        }
+      });
+    } catch (error) {
+      pollingRequestBotStatus(jobId, callback);
+    }
+  };
+
+  cy.request('post', '/api/v2/projects', params).then((res) => {
+    const { jobId } = res.body;
+    // install package can take a long time.
+    cy.wait(20000);
+    pollingRequestBotStatus(jobId, (result) => callback(result));
+  });
+});
```

Comment:
This should cache the result of the first creation and the use that bot as a template by copying files and performing and transforms on the files. I know we don't use this api in too many places, but I think it's worth investing in the performance here to not slow the e2e execution down significantly.

Reply:
I made some updates, before all spec start, will create a template bot. when execute each spec, will use it call save as API, and spec will get a new test bot.